tests/fafnir_tests/mod.rs

```diff
@@ -288,16 +288,19 @@ pub fn main_test(mut es_wrapper: ElasticSearchWrapper, pg_wrapper: PostgresWrapp
         &es_wrapper,
     );
 
-    // Test that the postgres wrapper contains 5 rows
-    let rows = &pg_wrapper.get_rows();
+    // Test that the postgres wrapper contains 5 rows in osm_poi_point
+    // and 2 rows in osm_poi_polygon
+    let rows = &pg_wrapper.get_rows(&"osm_poi_point");
     assert_eq!(rows.len(), 5);
-    // but the elastic search contains only 4 because the poi "poi too far" has not been loaded
```

Comment:
We shouldn't remove this comment. 
It seems this is still an expected behavior. An idea why it was not tested properly ?